### PR TITLE
voucher

### PR DESCRIPTION
**Pull request by Tran-Khac-Nam**
## Related Tickets:
- [#30183 ID](https://edu-redmine.sun-asterisk.vn/issues/30183)

## Notes Impacted Areas:
create + delete + show v

## Attachments:
- N/A.

### CODE REVIEW

**Comment by Tran-Khac-Nam** [comment]
ready

File: app/Http/Controllers/Supplier/VoucherController.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+class VoucherController extends Controller
+{
+    public function index()
+    {
+        $vouchers = Voucher::where('user_id', Auth::id())->orderBy('created_at', 'DESC')->get();
+
+        return view('supplier.voucher.index', compact('vouchers'));
+    }
+    public function destroy(Request $request)
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Supplier/VoucherController.php]:
```suggestion
    }
    
    public function destroy(Request $request)
```

File: app/Http/Controllers/Supplier/VoucherController.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    {
+        if (Voucher::create($request->all())) {
+            $data['success'] = trans('customer.success');
+            $data['msg'] = trans('customer.success_add_cart');
+
+            return response()->json($data, config('config.success'));
+        } else {
+            $data['msg'] = trans('customer.error_add_cart');
+            $data['error'] = trans('customer.error');
+        }
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Supplier/VoucherController.php]:
```suggestion
        }
            $data['msg'] = trans('customer.error_add_cart');
            $data['error'] = trans('customer.error');
        
```

**Comment by Tran-Khac-Nam** [comment]
ready

**Pull request merged**